src/ripple/basics/Log.h

```diff
@@ -236,6 +235,25 @@ deprecatedLogs()
     return logs;
 }
 
+class LogSquelcher
+{
+public:
+    LogSquelcher()
+        : severity_(deprecatedLogs().severity())
+    {
+        deprecatedLogs().severity(
+            beast::Journal::Severity::kNone);
+    }
+
+    ~LogSquelcher()
+    {
+        deprecatedLogs().severity(severity_);
+    }
+
+private:
+    beast::Journal::Severity const severity_;
+};
```

Comment:
There is zero benefit to not using `deprecatedLogs`. Once we no longer use the globals, such constructs will be unnecessary.